starter-project-web-vue2/components/aymen/BlogList.vue

```diff
@@ -45,6 +56,10 @@ export default {
   },
   methods: {
     ...mapActions('aymen', ['fetchBlogs', 'deleteBlog']),
+    signout() {
+      localStorage.removeItem("token")
+      this.$router.push('/aymen/signin')
```

Comment:
this could be replaced with `this.$auth.logout()` [link](https://auth.nuxtjs.org/api/auth/#logoutargs)